app/src/main/java/com/mapbox/dash/example/MainActivity.kt

```diff
@@ -450,19 +448,6 @@ class MainActivity : DrawerActivity() {
         }
     }
 
-    private fun etaPanelCustomization() {
-        bindSwitch(
-            switch = menuBinding.toggleTripProgress,
-            state = showTripProgress,
```

Comment:
`showTripProgress` flow should be removed

Reply:
Thanks. Fixed.